frontend/src/metabase/setup/selectors.ts

```diff
@@ -26,26 +29,42 @@ export const getInvite = (state: State): InviteInfo | undefined => {
   return state.setup.invite;
 };
 
-export const isLocaleLoaded = (state: State): boolean => {
+export const getIsLocaleLoaded = (state: State): boolean => {
   return state.setup.isLocaleLoaded;
 };
 
-export const isTrackingAllowed = (state: State): boolean => {
+export const getIsTrackingAllowed = (state: State): boolean => {
   return state.setup.isTrackingAllowed;
 };
 
-export const isStepActive = (state: State, step: number): boolean => {
+export const getIsStepActive = (state: State, step: number): boolean => {
   return getStep(state) === step;
 };
 
-export const isStepCompleted = (state: State, step: number): boolean => {
+export const getIsStepCompleted = (state: State, step: number): boolean => {
   return getStep(state) > step;
 };
 
-export const isSetupCompleted = (state: State): boolean => {
+export const getIsSetupCompleted = (state: State): boolean => {
   return getStep(state) === COMPLETED_STEP;
 };
 
 export const getDatabaseEngine = (state: State): string | undefined => {
   return getDatabase(state)?.engine || state.setup.databaseEngine;
 };
+
+export const getSetupToken = (state: State) => {
+  return getSetting(state, "setup-token");
+};
+
+export const getIsHosted = (state: State): boolean => {
+  return getSetting(state, "is-hosted?");
+};
```

Comment:
We need `is-hosted?` in a few different places, and also as a part of the "is paid plan" check. Would it make sense to move this selector to a shared module? Perhaps `metabase/selectors/settings`? Can be done later done